README.md

```diff
@@ -88,7 +93,7 @@ Which of the following represents a correct workflow for submitting a PR on a no
 Select 1:
 ```
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
-[] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[x] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
 [] git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git pull; create pull request
 ```
```

Comment:
Close, but it's the first answer. You want to make sure that you clone your forked repo to your computer. That's the one you'll be pushing your work too and later making a pull request from.